src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -1000,29 +1000,31 @@ private void DecodeInterceptsLocationAttribute(DecodeWellKnownAttributeArguments
                 return;
             }
 
-            var syntaxTrees = DeclaringCompilation.SyntaxTrees;
+            var referenceResolver = DeclaringCompilation.Options.SourceReferenceResolver ?? SourceFileResolver.Default;
```

Comment:
It's another case where we're basically repeating the decision. Think at least we should just have `SourceFileResolverOrDeafult`